### ISSUE TITLE
[TypeScript Error Interface ] Can't find correct error interface for cast error to this type (getAccessToken)

### ISSUE THREAD

**Issue by muradsofi** [auth0/nextjs-auth0#872]
### Checklist

- [X] The issue can be reproduced in the [sample app](https://github.com/auth0-samples/auth0-nextjs-samples/tree/main/Sample-01) (or N/A).
- [X] I have looked into the [README](https://github.com/auth0/nextjs-auth0#readme) and have not found a suitable solution or answer.
- [X] I have looked into the [examples](https://github.com/auth0/nextjs-auth0/blob/main/EXAMPLES.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/nextjs-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/nextjs-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com/c/sdks/5) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

When I try to use the `refresh token` to get a new `access token` via `getAccessToken` function my refresh token can be expired. Because that handling such cases I wrap  `getAccessToken` function call with try-catch. 

```
  try {
    const { accessToken } = await getAccessToken(req, res);
    console.log(accessToken)
  } catch (error) {
    console.log(JSON.stringify(error)); // {"error":"invalid_grant","error_description":"Unknown or invalid refresh token."}
  }
```

But the problem is I can't find any interface for cast error to this type. 

I know SDK has `AccessTokenError` but this error structure is not a similar error that I get 

https://auth0.github.io/nextjs-auth0/classes/utils_errors.accesstokenerror.html

### Reproduction

Request getAccessToken(req, res); with expired refresh token 

### SDK version

1.9.1

### Next.js version

10.2.3

### Node.js version

v16.13.0

**Comment by adamjmcgrath** [comment]
Hi @muradsofi - thanks for raising this.

We're doing some work on improving the errors in v2, so I'll make sure we include this issue.

For now, you'll just need to cast it as `{error: string, error_description: string} | AccessTokenError`

**Comment by adamjmcgrath** [comment]
Fixed in #878 - will go out in next beta release
